source/image-handler/image-request.js

```diff
@@ -46,9 +48,9 @@ class ImageRequest {
         const request = s3.getObject(imageLocation).promise();
         try {
             const originalImage = await request;
-            return Promise.resolve(originalImage.Body);
+            return Promise.resolve({ body: originalImage.Body, contentType: originalImage.ContentType });
```

Comment:
`s3.getObject` returns the ContentType (note the uppercase convention), we want to return in, with the body